README.md

```diff
@@ -21,6 +21,32 @@ infrastructure to study behavior of open source packages and to look for malicio
 We also hope that the components can be used independently, to provide package feeds or runtime
 behavior data for anyone interested.
 
+# Configuration
+
+A YAML configuration file can be provided with the following format:
+
+```
+enabled_feeds:
+- pypi
+- npm
+- goproxy
+- rubygems
+- crates
+
+publisher:
+  type: 'gcp_pubsub'
+  config:
+    url: "gcppubsub://foobar.com"
+
+http_port: 8080
+```
+
+To specify this configuration file, define its path in your environment under the `PACKAGE_FEEDS_CONFIG_PATH` variable.
+
+## Legacy Configuration
+
+Legacy configuration methods are still supported. To configure the application with all feeds enabled, the environment variable `OSSMALWARE_TOPIC_URL` can be used to select the GCP pubsub publisher and `PORT` will configure the port for the HTTP server.
```

Comment:
This reads a little off for me, maybe something like `By default, if no configuration file is provided all feeds will be enabled`

Reply:
Reworded to clarify this